### DISCUSSION TITLE
Dynamic content picker

### DISCUSSION THREAD

**Discussion by konika-jindal** [OrchardCMS/OrchardCore#16518] [Q&A]
I have one  content type named policy , in that i have two content pickers named jurisdiction and references , now in jurisdiction i have reference to content type jurisdiction , now in reference it has reference to policy and i want to have only that policies in dropdown which have same jurisdiction i.e. dynamic content picker . How can i achieve that . Please give solution asap

**Comment by Piedone** [comment]
You can configure the content type under each content picker:

https://github.com/user-attachments/assets/e664df18-e659-425d-8ea3-a4ef4dca3549

**Comment by konika-jindal** [comment]
Yes , For eg In articles content picker  you chose article content type , it will show all articles in its dropdown . But suppose you have one content picker in article let say country , you have to choose country for each article . Now , i am using articles content picker(having reference to article content type )  in article content type , now i have one article having country UK , then I want only those articles in articles content picker dropdown which have country UK , i don't want all articles in its dropdown


Please let me know how can I achieve that?

**Reply by Piedone** [reply]
You can't easily do such custom filtering currently. The content picker searches for the display text though, so you can utilize that for filtering if suitable.

**Comment by konika-jindal** [comment]
Is there any possibility to do that within cms?

**Reply by Piedone** [reply]
Please reply under the given message, don't open a new one, because this gets hard to follow.

As I said, no, this is not possible now. You can create your own field based on the content picker that has this feature, and add filtering specific to your content types there (filtering and search otherwise are completely supported by the data model)
